src/main/java/uk/gov/di/ipv/cri/common/library/persistence/item/SessionItem.java

```diff
@@ -24,6 +24,10 @@ public class SessionItem {
     private String subject;
     private String persistentSessionId;
     private String clientSessionId;
+    private String latestDcsResponseResourceId;
+    private String userId;
+    private String govukSigninJourneyId;
```

Comment:
This field can be removed as the data is already assigned to the `clientSessionId` field